google-cloud-compute-v1/samples/README.md

```diff
@@ -0,0 +1,17 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google
+Cloud Platform logo" title="Google Cloud Platform" align="right" height="96"
+width="96"/>
+
+# Google Cloud Compute API Samples
+
+## Description
+
+These samples show how to use the [Compute Engine API](https://cloud.google.com/compute/docs/reference/rest/v1).
```

Comment:
```suggestion
These samples show how to use the [Ruby API Client library](https://github.com/googleapis/google-cloud-ruby/tree/master/google-cloud-compute-v1) for the [Google Cloud Compute Engine API](https://cloud.google.com/compute/docs/reference/rest/v1)
```

Reply:
Instead of `[Ruby API Client library]`, recommend using `[Ruby Cloud Client Library]` to better distinguish from the client library in the googleapis/google-api-ruby-client repo